src/modules/dashboard/components/CoinMachine/CoinMachine.tsx

```diff
@@ -100,17 +103,32 @@ const CoinMachine = ({
     <div className={styles.main}>
       <BreadCrumb elements={breadCrumbs} />
       <div className={styles.grid}>
-        <div className={styles.buy}>
-          <div className={styles.filler}>Buy Tokens</div>
-        </div>
-        <div className={styles.timeRemaining}>
-          <div className={styles.filler}>Time Remaining</div>
-        </div>
-        <div className={styles.tokensRemaining}>
-          <div className={styles.filler}>Tokens Remaining</div>
-        </div>
+        {(saleStarted && (
+          <div className={styles.saleStarted}>
+            <SaleStateWidget
+              state={SaleState.Success}
+              price="1234234340000000"
+              amount="123423434000000000000"
+              timeLeftToNextSale={864000}
+              nativeToken={{ symbol: 'CLNY', decimals: 18, name: 'CLNY' }}
+              transactionToken={{ symbol: 'ETH', decimals: 18, name: 'ETH' }}
```

Comment:
Just a heads up that these tokens might not actually be the colony's native token or ETH/XDAI. They can be _any_ token the colony has.

So I guess, better names for these props could be `soldToken` and `receivedToken`

_(this is just a nit-pick I noticed while rebasing on your changes, it's not something major, so you can change it when you wire in the logic)_

Reply:
Never mind, I fixed it in my #2445 

![Screenshot from 2021-07-27 19-30-33](https://user-images.githubusercontent.com/1193222/127192247-560632cf-b52e-4ae0-9af7-76b489d209a5.png)